headers/functions/overlay31.h

```diff
@@ -7,6 +7,7 @@ void DungeonMenuSwitch(int idx);
 void CreateStairsMenuState(struct entity* entity);
 void StairsSubheadingCallback(int window_id);
 int HandleStairsMenu(void);
+bool EntityIsValid(struct entity* entity);
```

Comment:
Yes this is going to cause problems for c-of-time. See `EntityIsValidMoveEffects`, this should similarly have a unique name

Reply:
Renamed to `EntityIsValidOverlay31`.